dtclient/client.go

```diff
@@ -76,8 +76,9 @@ const (
 
 // Known flavors.
 const (
-	FlavorDefault = "default"
-	FlavorMUSL    = "musl"
+	FlavorDefault     = "default"
+	FlavorMUSL        = "musl"
```

Comment:
Do we still need those consts?

Reply:
Currently unused. But they are still valid arguments to `GetLatestAgent`.